Zotero/Scenes/Detail/Items/ViewModels/ItemsActionHandler.swift

```diff
@@ -96,8 +96,8 @@ struct ItemsActionHandler: ViewModelActionHandler, BackgroundDbProcessingActionH
                 state.error = .dataLoading
             }
 
-        case .saveNote(let key, let text, let tags):
-            self.saveNote(text: text, tags: tags, key: key, in: viewModel)
+        case .processNoteSaveResult(let result):
+            self.processNoteSaveResult(result: result, in: viewModel)
```

Comment:
I guess since you're already editing this, always adhere to our new rules, so remove `self` here too. I know the rest of this class will have `self`s, but we'll tackle it one by one or at least there will be less refactoring later when going through the whole class.

Reply:
Check https://github.com/zotero/zotero-ios/pull/826/commits/01c985e8f05f204526ffaba59aa4719e745f2f88